index.html

```diff
@@ -2,377 +2,405 @@
 <html lang="en">
```

Comment:
- [ ] The width of the content of the website is wider than the width of the viewport. So it scrolls horizontally. Please make sure the width of every element in the website doesn't exceed the size of the viewport. I think you should use `vw` to set the width of elements. 

![Screen Shot 2022-07-08 at 23 05 07](https://user-images.githubusercontent.com/12524453/178062883-aab27420-1f26-459f-b41b-0e11b182546d.png)

Reply:
- Width managed and horizontal scroller now, no horizontal scroll trough large width